lib/jekyll/converters/scss.rb

```diff
@@ -75,11 +75,28 @@ def sass_dir_relative_to_site_source
       end
 
       def sass_load_paths
+        paths = user_sass_load_paths + [sass_dir_relative_to_site_source]
+
         if safe?
-          [sass_dir_relative_to_site_source]
-        else
-          (user_sass_load_paths + [sass_dir_relative_to_site_source]).uniq
-        end.select { |load_path| File.directory?(load_path) }
+          # Sanitize paths to prevent any attack vectors (.e.g. `/**/*`)
+          paths.map! { |path| Jekyll.sanitized_path(@config["source"], path) }
+        end
+
+        # Expand file globs (e.g. `node_modules/*/node_modules` )
+        Dir.chdir(@config["source"]) do
```

Comment:
@benbalter `@config["source"]`? [It comes from `site.config`.](https://github.com/jekyll/jekyll/blob/v3.2.1/lib/jekyll/converter.rb#L31-L36)